### DISCUSSION TITLE
UnRoot pastas no Linux

### DISCUSSION THREAD

**Discussion by ArtumosGOC** [codaqui/institucional#177] [Geral]
# Estava tendo este problema, mas aprendi resolve-lo!
## UnRoot pastas no Linux Imagens
![image](https://github.com/codaqui/institucional/assets/63540372/9fcf20c1-13e3-4a99-a2ef-c802c62e96f1)

![image](https://github.com/codaqui/institucional/assets/63540372/1e263aa5-e8d1-4791-b1c3-ba79bd30f540)

![image](https://github.com/codaqui/institucional/assets/63540372/e363048d-970c-4d5a-9b0c-190185d2b1ec)
> Basta utilizar os comandos a baixo

## Command Shell

```console
sudo chown -R [USUARIO] [DIRETORIO]
```

**Comment by endersonmenezes** [comment]
Boa @ArtumosGOC !

Olha aqui esse site: https://phoenixnap.com/kb/linux-chown-command-with-examples

Podia traduzir ele e montar uma postagem pro Blog né?

**Reply by ArtumosGOC** [reply]
Consigo sim Prof

**Reply by ArtumosGOC** [reply]
@endersonmenezes

**Reply by endersonmenezes** [reply]
Pode criar como postagem do repositório tá bem legal! Categoriza como tutorial comandos linux
